### PR TITLE
pr webhook test

### PR DESCRIPTION
**Pull request by haya14busa**

### CODE REVIEW

File: github.go [hunk 1]
```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/google/go-github/github"
 )
 
+var X = 1
```
Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var X should have comment or be unexported

File: github.go [hunk 2]
```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/google/go-github/github"
 )
 
+var XY = 1
```
Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var XY should have comment or be unexported

File: github.go [hunk 3]
```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/google/go-github/github"
 )
 
+var XYZ = 1
```
Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var XYZ should have comment or be unexported

File: github.go [hunk 4]
```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/google/go-github/github"
 )
 
+var XYZ = 2
```
Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var XYZ should have comment or be unexported

Comment (haya14busa) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var XYZ should have comment or be unexported

Comment (haya14busa) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var XYZ should have comment or be unexported

Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var XYZ should have comment or be unexported

File: github.go [hunk 5]
```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/google/go-github/github"
 )
 
+var XYZ = 3
```
Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var XYZ should have comment or be unexported

**Pull request closed without merging**